source/beamline/SXRMB/SXRMBBeamline.cpp

```diff
@@ -642,19 +642,22 @@ void SXRMBBeamline::setupSampleStage()
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageZ_);
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageR_);
 
-	// Ambiant Endstation sample state
+	// Ambiant Endstation sample stage
+	ambiantTableHeight_ = new AMPVwStatusControl("Ambiant Table Height", "TBL0000-E07-01:Y:mm:fbk", "TBL0000-E07-01:Y:mm", "TBL0000-E07-01:Y:mm:status", QString(), this, 0.005, 2.0, new CLSMAXvControlStatusChecker());
```

Comment:
There appears to be no stop pv in here, where there is in the others. I assume, in my ignorance, that this is by design? 

(this is much a question for my own curiosity)